src/LensPlatformClient.ts

```diff
@@ -19,6 +19,11 @@ export interface LensPlatformClientOptions {
   exceptionHandler?: (exception: unknown) => void;
 }
 
+// we need to export this so
+const SDK_HTTPError extends Error {
```

Comment:
Wait, is this valid TypeScript?

Reply:
yeah, should be class, I use Github editor 🥇

Reply:
I believe that's what was meant:
```suggestion
class SDK_HTTPError extends Error {
```

Having class-based exceptions is a great idea - it will make guarding against these errors way easier.